.ci/cico_build_and_deploy_custom_rh-che.sh

```diff
@@ -62,33 +57,34 @@ echo "***======================================***"
 echo "Running ${JOB_NAME} build number #${BUILD_NUMBER}, testing creds:"
 
 CREDS_NOT_SET="false"
-#curl ${ORIGIN_CLIENTS_URL} > origin-clients.rpm
-#yum install --assumeyes ./origin-clients.rpm
-curl "https://mirror.openshift.com/pub/openshift-v3/clients/${OC_VERSION}/linux/oc.tar.gz" | tar xvz -C /usr/local/bin
-if [ -z ${RH_CHE_AUTOMATION_DOCKERHUB_USERNAME} ] || [ -z ${RH_CHE_AUTOMATION_DOCKERHUB_PASSWORD} ]; then
-  echo "Dockerhub credentials not set"
+curl -s "https://mirror.openshift.com/pub/openshift-v3/clients/${OC_VERSION}/linux/oc.tar.gz" | tar xvz -C /usr/local/bin
+if [ -z "${DEVSHIFT_USERNAME}" ] || [ -z "${DEVSHIFT_PASSWORD}" ]; then
+  echo "Docker registry credentials not set"
   CREDS_NOT_SET="true"
 fi
-if [ -z ${RH_CHE_AUTOMATION_RDU2C_USERNAME} ] || [ -z ${RH_CHE_AUTOMATION_RDU2C_PASSWORD} ]; then
+if [ -z "${RH_CHE_AUTOMATION_RDU2C_USERNAME}" ] || [ -z "${RH_CHE_AUTOMATION_RDU2C_PASSWORD}" ]; then
   echo "RDU2C credentials not set"
   CREDS_NOT_SET="true"
 else
-  oc login ${OPENSHIFT_RDU2C_URL} --insecure-skip-tls-verify \
-                                  -u ${RH_CHE_AUTOMATION_RDU2C_USERNAME} \
-                                  -p ${RH_CHE_AUTOMATION_RDU2C_PASSWORD} && echo "Credentials test OK" || {
+  if oc login ${DEV_CLUSTER_URL} --insecure-skip-tls-verify \
```

Comment:
We should avoid nested `if` as much as possible. And we should fail-fast too. So I would suggest instead of:
```bash
if [ somecondition ]; then
  echo "some error output"
  CREDS_NOT_SET="true"
else
   if [ anothercondition ]; then
       .....
   fi
fi

if [ "${CREDS_NOT_SET}" == "true" ]; then
   exit 2
fi
```

to 

```bash
if [ somecondition ]; then
  echo "some error output"; exit 2
fi

if [ anothercondition ]; then
   .....
fi
```